egs/wsj/s5/steps/nnet3/xconfig_to_configs.py

```diff
@@ -28,6 +28,12 @@ def get_args():
         epilog='Search egs/*/*/local/{nnet3,chain}/*sh for examples')
     parser.add_argument('--xconfig-file', required=True,
                         help='Filename of input xconfig file')
+    parser.add_argument('--existing-model',
```

Comment:
The doc-string doesn't explain what this option expects, which is presumably a filename of a .mdl
file containing a previously trained neural net.